core/modules/parsers/wikiparser/wikiparser.js

```diff
@@ -165,6 +165,17 @@ WikiParser.prototype.findNextMatch = function(rules,startPos) {
 	return matchingRule;
 };
 
+WikiParser.prototype.parseRule = function(rule) {
+	var start = this.pos;
+	var blocks = rule.parse();
+	for(var i=0; i<blocks.length; i++) {
+		var block = blocks[i];
+		if (block.start !== undefined) block.start = start;
```

Comment:
`if` statements should not have whitespace between the `if` and the opening parenthesis. We also use braces around the body.